client/app/telecom/pack/xdsl/modem/router/dhcp/translations/Messages_fr_FR.xml

```diff
@@ -23,4 +23,7 @@
    <translation id="xdsl_modem_dhcp_success" qtlid="311947">Le DHCP <strong>{{ name }}</strong> a été édité avec succès!</translation>
    <translation id="xdsl_modem_dhcp_submit_error" qtlid="385273">Oups ! Nous n'avons pas pu mettre à jour le DHCP.</translation>
    <translation id="xdsl_modem_dhcp_error" qtlid="385286">Oups ! Nous n'avons pas pu récupérer les informations relatives au DHCP.</translation>
+   <translation id="xdsl_modem_dhcp_modal_title">Attention IP refusée</translation>
```

Comment:
You should see with the Content team for the messages, this doesn't respect their guidelines :)